packages/clay-css/src/scss/cadmin/variables/_alerts.scss

```diff
@@ -267,12 +267,12 @@ $cadmin-alert-notification-sm-down: map-merge(
 $cadmin-alert-autofit-row: () !default;
 $cadmin-alert-autofit-row: map-deep-merge(
 	(
-		margin-left: -4px,
-		margin-right: -4px,
+		margin-left: -8px,
+		margin-right: -8px,
 		width: auto,
 		autofit-col: (
-			padding-left: 4px,
-			padding-right: 4px,
+			padding-left: 8px,
+			padding-right: 8px,
```

Comment:
@pat270 this `8px` increase can affect the spacing between `alert-indicator` and the text which is 8px, in this case, I think we will have `16px` spacing.

Reply:
@matuzalemsteles good catch, I just updated this.